m2cgen/ast.py

```diff
@@ -237,7 +249,7 @@ def __str__(self):
     (PowExpr, lambda e: [e.base_expr, e.exp_expr]),
     (VectorVal, lambda e: e.exprs),
     (IfExpr, lambda e: [e.test, e.body, e.orelse]),
-    ((ExpExpr, TanhExpr, TransparentExpr), lambda e: [e.expr]),
+    ((ExpExpr, SqrtExpr, TanhExpr, TransparentExpr), lambda e: [e.expr]),
```

Comment:
Do we need to include `SqrtExpr` into one of the tests that we have for `count_exprs`?

Reply:
Sure! Forgot about it, sorry!